manifest.md

```diff
@@ -148,11 +148,10 @@ The image manifest provides a configuration and a set of layers for a container
 
 - **`config`** *object*
 
-    The config field references a configuration object for a container, by digest.
-    This configuration item is a JSON blob that the runtime uses to set up the container.
-    This new schema uses a tweaked version of this configuration to allow image content-addressability on the daemon side.
```

Comment:
Notes on “tweaked” [here][1], so it should be safe to remove.

[1]: http://ircbot.wl.linuxfoundation.org/eavesdrop/%23opencontainers/%23opencontainers.2016-05-07.log.html#t2016-05-07T00:29:41